gph/python/test/test_ripser.py

```diff
@@ -311,3 +311,18 @@ def test_gens_order_vertices_higher_dimension():
 
     assert len(gens_fin_dim2) == 1
     assert np.array_equal(gens_fin_dim2[0], np.array([1, 0, 5, 4]))
+
+
+def test_ph_maxdim_0():
+    """Regression test for issue #39, an issue was found when only computing
+    up to dimension 0. The test also compares the results of dimension 0
+    when maxdim=1 and maxdim=0"""
+    X = np.array([[1., 2], [3, 4], [5, 0]])
+    res_maxdim_0 = ripser(X, maxdim=0)['dgms'][0]
+    res_maxdim_1 = ripser(X, maxdim=1)['dgms'][0]
+
+    # Verifies that the number of barcodes is the same
+    assert res_maxdim_0.shape[0] == res_maxdim_1.shape[0]
```

Comment:
Redundant check now, you can remove it :)

Reply:
Done